resources/schema.py

```diff
@@ -948,6 +1330,14 @@ def resolve_winter_storage_areas(self, info, **kwargs):
         ).select_related("availability_level", "municipality")
 
 
+IMAGE_FILE_DESCRIPTION = (
+    "The `imageFile` field takes an image as input. "
+    "To provide the file, you have to perform the request with a client "
+    "that conforms to the [GraphQL Multipart Request Spec]"
+    "(https://github.com/jaydenseric/graphql-multipart-request-spec)."
+)
```

Comment:
Let's also get rid of this since we don't use the image upload anymore.

Reply:
fixed